sources/packages/web/src/services/DesignationAgreementService.ts

```diff
@@ -55,19 +57,23 @@ export class DesignationAgreementService {
 
   async updateDesignationAgreement(
     designationId: number,
-    designation: UpdateDesignationDetailsAPIInDTO,
+    designation: UpdateDesignationDetailsModel,
   ): Promise<void> {
-    /**Filtering the locations which are either approved or denied(already approved location) only.
-     * locationsDesignations will have value only when approval is done.
-     */
+    // Filtering the locations which are either approved or denied(already approved location) only.
+    // locationsDesignations will have value only when approval is done.
     designation.locationsDesignations =
       designation.locationsDesignations?.filter(
         (location) =>
           location.existingDesignationLocation || location.approved === true,
       );
+    const { excludeExtraneousValues } = useFormioUtils();
+    const typedPayload = excludeExtraneousValues(
+      UpdateDesignationDetailsAPIInDTO,
+      designation,
+    );
     await ApiClient.DesignationAgreement.updateDesignationAgreement(
       designationId,
-      designation,
+      typedPayload,
```

Comment:
In my local, my getting below error, 
![image](https://github.com/bcgov/SIMS/assets/77353155/38bccfd2-7077-4e7c-8948-80b19467791a)